tests/infra_test.go

```diff
@@ -95,6 +99,83 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 		}
 	})
 
+	Describe("changes to the kubernetes client", func() {
+
+		BeforeEach(func() {
+			tests.BeforeTestCleanup()
+		})
+
+		It("on the controller rate limiter should lead to delayed VMI starts", func() {
+			By("first getting the basetime for a replicaset")
+			replicaset := tests.NewRandomReplicaSetFromVMI(libvmi.NewCirros(libvmi.WithResourceMemory("1Mi")), int32(0))
+			replicaset, err = virtClient.ReplicaSet(util.NamespaceTestDefault).Create(replicaset)
+			Expect(err).ToNot(HaveOccurred())
+			start := time.Now()
+			libreplicaset.DoScaleWithScaleSubresource(virtClient, replicaset.Name, 10)
+			fastDuration := time.Now().Sub(start)
+			libreplicaset.DoScaleWithScaleSubresource(virtClient, replicaset.Name, 0)
+
+			By("reducing the throughput on controller")
+			originalKubeVirt := util.GetCurrentKv(virtClient)
+			originalKubeVirt.Spec.Configuration.ControllerConfiguration = &v1.ReloadableComponentConfiguration{
+				RestClient: &v1.RESTClientConfiguration{
+					RateLimiter: &v1.RateLimiter{
+						TokenBucketRateLimiter: &v1.TokenBucketRateLimiter{
+							Burst: 3,
```

Comment:
Why you set Bust=3 and QPS=2 here, and in the handler test Bust=1 and QPS=1?
Can this test also use Bust=1 and QPS=1?

Reply:
If I set it too low we have the issue that the leader-election will not work either and we have virt-controllers crash-looping.

Reply:
In this case, can we use Bust=3 and QPS=2 for all small cases?
We could also include a comment that this is the minimal requirement to make sure that leader-election will work.
So, anyone can understand that in the future.